src/common/actions/repositories.js

```diff
@@ -5,8 +5,10 @@ import { conf } from '../helpers/config';
 const BASE_URL = conf.get('BASE_URL');
 
 export const FETCH_REPOSITORIES = 'FETCH_REPOSITORIES';
+export const FETCH_REPOSITORIES_SUCCESS = 'FETCH_REPOSITORIES_SUCCESS';
```

Comment:
I've split `FETCH_REPOSITORIES_SUCCESS` out from `SET_REPOSITORIES` so that we can clear repositories when the page is changed without setting the success status too early. Setting success with the repositories is a side-effect - setting them doesn't necessarily mean data has been loaded from service.